package.json

```diff
@@ -91,11 +91,12 @@
   "scripts": {
     "build:evergreen": "node .evergreen/generate_evergreen_tasks.js",
     "build:ts": "rimraf lib && tsc",
-    "build:dts": "npm run build:ts && api-extractor run --typescript-compiler-folder node_modules/typescript --local && rimraf 'lib/**/*.d.ts*'",
+    "build:dts": "npm run build:ts && api-extractor run && rimraf 'lib/**/*.d.ts*'",
     "build:docs": "npm run build:dts && typedoc",
     "check:bench": "node test/benchmarks/driverBench",
     "check:coverage": "nyc npm run check:test",
-    "check:lint": "npm run check:ts && eslint -v && eslint --max-warnings=0 --ext '.js,.ts' src test",
+    "check:lint": "npm run build:dts && npm run check:eslint",
+    "check:eslint": "eslint -v && eslint --max-warnings=0 --ext '.js,.ts' src test",
```

Comment:
@emadum @mbroadst 
I figure we should add api-extractor to our linting now if we want to maintain access modifiers going forward, does this seem ok to y'all? If we think its overkill for now, I can roll this back